docker-compose.default.yml

```diff
@@ -17,6 +17,13 @@ services:
     environment:
       - password=P4ssW0rD
 
+frontend: # container for python frontend
+  build: frontend/.
+  command: uwsgi uwsgi.ini
+  volumes:
+    - "./config/frontend/config.yml:/code/config.yml:ro"
+    - "./config/frontend/uwsgi.ini:/code/uwsgi.ini:ro"
```

Comment:
the indentation of this doesn't really look correct to me.

It's supposed to be a service (same indentation as the `auth` container above or `nginx-rtmp` container below)

What kind of errors are you getting with the unmodified file? Maybe you missed to copy the default configuration for the frontend?

I just tested from master with the following commands and it did work on my machine:
´´´bash
cp frontend/config.example.yml frontend/config.yml
docker-compose -f docker-compose.default.yml build
docker-compose -f docker-compose.default.yml up
´´´